ds/settings.sh

```diff
@@ -26,3 +26,9 @@ DBPORT=3306
 DBNAME=lbd
 DBUSER=lbd
 DBPASS=lbd
+
+# If you want to use Redis, put its hostname or IP address here
+# Else you can comment out this variable or set as empty.
+# Default value is 'redis'.
```

Comment:
Use commenting style consistent with the rest of the script: `### `
This is not magic or better, but just to be consistent.